docs/chapters/baseplatform/building-PELUX-sources.rst

```diff
@@ -8,10 +8,9 @@ Building PELUX sources
 This chapter details how to download and configure the sources of a PELUX build, so
 that an image can be built.
 
-To obtain a PELUX release, simply download the manifest (`pelux.xml`) used for
-building, and decide what image to build. Currently there are two versions:
-`core-image-pelux-minimal` and `core-image-pelux-qtauto-neptune`. The latter
-being a version that includes `Qt Automotive Suite`_ components that enable the
+Currently there are two versions of PELUX:`core-image-pelux-minimal` and
```

Comment:
There are more than two versions of PELUX. Referring to the SDK as well as to the table.

Reply:
Has the table been updated since this comment was made? I can't see "sdk" in the table.
Maybe it should be included or added, since it's not in the table.

Reply:
Better to refer to the table below and be more generic in the text.

Reply:
I resolved this by removing the paragraph. It doesn't add any information that isn't available in the table further down in the document. Future improvements and clarifications might be needed, but the text as it was was confusing.